scripts/apps/authoring/widgets/widgets.ts

```diff
@@ -229,6 +252,51 @@ function WidgetsManagerCtrl(
         }
     };
 
+    $scope.pinWidget = (widget: IWidget) => {
+        if ($scope.pinnedWidget) {
+            $scope.pinnedWidget.pinned = false;
+        }
+
+        if (!superdesk.pinnedWidgetResized && widget && !$scope.pinnedWidget) {
```

Comment:
Can you use a local variable in the file for  `pinnedWidgetResized` instead of setting it on `superdesk`?